digiroad2-oracle/src/main/resources/db/migration/V193__date_period_table_and_properties.sql

```diff
@@ -0,0 +1,25 @@
+create table date_period_value (
+	id number primary key,
+	asset_id number references asset not null,
+	property_id references property not null,
+	start_date TIMESTAMP not null,
+	end_date TIMESTAMP not null
+);
+
+insert into localized_string (id, value_fi, value_sv, created_by, created_date)
+values (primary_key_seq.nextval, 'Kelirikkokausi', NULL, 'db_migration_v193', sysdate);
+
+insert into property(id, asset_type_id, property_type, required, created_by, public_id, name_localized_string_id)
+values (primary_key_seq.nextval, 130, 'date_period', '0', 'db_migration_v193', 'spring_thaw_period', (select id from localized_string where value_fi = 'Kelirikkokausi' and created_by = 'db_migration_v193'));
+
+insert into localized_string (id, value_fi, value_sv, created_by, created_date)
+values (primary_key_seq.nextval, 'Jokavuotinen', NULL, 'db_migration_v193', sysdate);
+
+insert into property(id, asset_type_id, property_type, required, created_by, public_id, name_localized_string_id)
+values (primary_key_seq.nextval, 130, 'checkbox', '0', 'db_migration_v193', 'annual_repetition', (select id from localized_string where value_fi = 'Jokavuotinen' and created_by = 'db_migration_v193'));
```

Comment:
The public ID on property table should be in Finnish

Reply:
I don't disagree, just wondering why. (we have public ids in both english and finnish)

Reply:
let's keep it with English version like as we talk.